### PR TITLE
Metadata Update

### PR DESCRIPTION
**Pull request by jzanecook**
See commit descriptions for more information.

### CODE REVIEW

File: agent/js/README.md [hunk 1]
```diff
@@ -37,5 +40,84 @@ const taskHandler = async (
...
+    "input": "task-input-to-your-agent"
+  }'
+```
+
+You will get a response like this:
+
+```json
+{
+  "input": "task-input-to-your-agent",
+  "task_id": e6d768bb-4c50-4007-9853-aeffb46c77be",
```
Comment (mlejva) [inline agent/js/README.md:96]: Please add a missing leading quote

**Comment by mlejva** [comment]
Looks good to me! Let's wait either for @ValentaTomas or @jakubno and merge this if they don't have any issues with it

File: agent/js/package.json [hunk 2]
```diff
@@ -29,11 +39,28 @@
...
     "require": "./dist/index.js",
-    "default": "./dist/index.modern.js",
     "types": "./dist/index.d.ts"
   },
   "files": [
-    "dist"
+    "dist",
+    "README.md",
+    "package.json",
+    "package-lock.json"
```
Comment (jakubno) [inline agent/js/package.json:48]: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#files

`readme.md` and `package.json` are always included,  but let's keep them here, it's better to be explicit. `package-lock.json` can't be added, so remove it, to prevent some potential confusion

File: agent/js/package.json [hunk 3]
```diff
@@ -29,11 +39,28 @@
...
+  ],
+  "keywords": [
```
Comment (jakubno) [inline agent/js/package.json:50]: not sure about the keywords, lot of them seems irrelevant to me. I would go with something like this: 
- agent
- agent communication
- agent protocol
- agent standard
- agent interaction
- agent wrapper
- agent api

WDYT @ValentaTomas?

**Review by jakubno — COMMENTED** [review]
Thanks, nice improvements! 💯

I will add release pipeline in a bit so we can publish 🎉

File: agent/js/package.json (hunk 3 above)
Comment (ValentaTomas) [reply agent/js/package.json:50]: I generally try to add keywords about what the package is using (like `javascript`, `nodejs`) and what the package is solving (`api`, `agent-protocol`, ...).
Very specific things like `task` are probably too general and you wouldn't know about them if you didn't know about the package already.

The best way to look at this is "if I'm developer using X and I want to handle X, what are the keywords I would use in the search query"?

The keywords should also be without spaces - `agent-api`, etc.

File: agent/js/package.json [hunk 4]
```diff
@@ -2,9 +2,19 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/e2b-dev/agent-protocol"
+  },
   "main": "./dist/index.js",
-  "module": "./dist/index.m.js",
-  "unpkg": "dist/index.umd.js",
   "types": "dist/index.d.ts",
   "scripts": {
     "prepublishOnly": "pnpm build",
```
Comment (ValentaTomas) [inline agent/js/package.json:20]: I think the pnpm here should be npm?

File: agent/js/package.json [hunk 5]
```diff
@@ -29,11 +39,28 @@
   },
   "exports": {
     "require": "./dist/index.js",
```
Comment (ValentaTomas) [inline agent/js/package.json:41]: I'm not sure if the `require` here is necessary - the exports from the package should be handled by the top level fields

```json
"main": "dist/cjs/index.js",
"module": "dist/esm/index.js",
"umd": "dist/umd/index.js",
```

if the build process correctly outputs them.

File: agent/js/package.json [hunk 6]
```diff
@@ -2,9 +2,19 @@
...
+  "license": "MIT",
+  "author": {
+    "name": "e2b",
+    "email": "hello@e2b.dev",
+    "url": "https://e2b.dev"
+  },
+  "bugs": "https://github.com/e2b-dev/agent-protocol/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/e2b-dev/agent-protocol"
```
Comment (ValentaTomas) [inline agent/js/package.json:15]: We want the url here to reflect the subdirectory with the package.

File: agent/js/README.md [hunk 7]
```diff
@@ -18,12 +21,12 @@ import Agent, {
 } from "agent-protocol";
 
 const taskHandler = async (
```
Comment (ValentaTomas) [inline agent/js/README.md:23]: I generally prefer the functions to be declared as
```
function foo() {}
```
and not anonymous functions assigned to a variable, but this is a minor thing.

File: agent/js/package.json (hunk 3 above)
Comment (jzanecook) [reply agent/js/package.json:50]: You can thank ChatGPT for that list of keywords lol

Comment (ValentaTomas) [reply agent/js/package.json:50]: 🤔

Comment (jzanecook) [reply agent/js/package.json:50]: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#homepage

So considering that the keywords are just there for searching when using things like `npm search ...` you would rather it be more complex strings like "agent-api" or "agent-protocol"?

Trimming it down, I'm looking at this right now:

```
  "keywords": [
    "agent",
    "communication",
    "protocol",
    "agent-protocol",
    "agent-api",
    "nodejs",
    "javascript",
    "typescript",
    "sdk",
    "api",
    "wrapper"
  ],
```

File: agent/js/package.json (hunk 4 above)
Comment (jzanecook) [reply agent/js/package.json:20]: I had just been using what was in the repo since the start. If you want it to be npm it can be, although I believe pnpm is slightly faster from some discussions I've had. Up to you!

File: agent/js/package.json (hunk 5 above)
Comment (jzanecook) [reply agent/js/package.json:41]: The build process does not output them. I had issues with the tsup build process before which is why I had the other process, but I haven't experimented with this one to create those outputs.

File: agent/js/package.json (hunk 6 above)
Comment (jzanecook) [reply agent/js/package.json:15]: Alright, that's changed to `https://github.com/e2b-dev/agent-protocol/tree/main/agent/js`.

File: agent/js/README.md (hunk 7 above)
Comment (jzanecook) [reply agent/js/README.md:23]: I can change that in the example if you'd like.

Comment (jzanecook) [reply agent/js/README.md]: I changed it in the example and readme.

File: agent/js/package.json (hunk 4 above)
Comment (jzanecook) [reply agent/js/package.json]: I saw the GH action (#11) and figured I'd go ahead and change it to npm. :+1:

File: agent/js/package.json (hunk 5 above)
Comment (jakubno) [reply agent/js/package.json:41]: Do we even need the `exports` section?

Comment (jzanecook) [reply agent/js/package.json:41]: :+1:

**Review by jakubno — APPROVED** [review]
